Repositories/HumanRepository.cs

```diff
@@ -1,37 +1,59 @@
-﻿namespace DataManager.Repositories
+﻿using DataManager.Models;
+
+
+namespace DataManager.Repositories
 {
-    // TODO: zrób odzielna klase gdzie zaMOCK'ujesz dane (zrobisz sztuczne dane np jakas liste z Humanami)
+    // DONE: zrób odzielna klase gdzie zaMOCK'ujesz dane (zrobisz sztuczne dane np jakas liste z Humanami)
     public class HumanRepository : IHumanRepository
     {
-        // TODO: dodaj humana do listy
-        public Human CreateHuman(Human human)
+        private readonly List<Human> _humans = new List<Human>();
+
+        // DONE: pobierz wszystkich humanów ( cala liste )
+        public ICollection<Human> GetHumans()
         {
-            throw new NotImplementedException();
-        }
+            return _humans;
 
+        }
 
-        // TODO: usun humana z listy
-        public void DeleteHumanById(int id)
+        // DONE: dodaj humana do listy
+        public Human AddHuman(Human human)
         {
-            throw new NotImplementedException();
+            _humans.Add(human);
+            return human;
         }
 
-        // TODO: pobierz humana z listy po id
-        public Human GetHumanById(int id)
+        // DONE: usun humana z listy
+        public void DeleteHumanById(int id)
         {
-            throw new NotImplementedException();
+            var human = GetHumanById(id);
+            if (human != null)
+            {
+                _humans.Remove(human);
+            }
         }
 
-        // TODO: pobierz wszystkich humanów ( cala liste )
-        public ICollection<Human> GetHumans()
+        // DONE: pobierz humana z listy po id
+        public Human GetHumanById(int id)
         {
-            throw new NotImplementedException();
+            return _humans.FirstOrDefault(h => h.Id == id);
         }
 
-        // TODO: zaktualizuj rekord gdzie id == twojemu wrzuconemu
+        // DONE: zaktualizuj rekord gdzie id == twojemu wrzuconemu
+        // 
         public Human UpdateHuman(Human human)
         {
-            throw new NotImplementedException();
+            var existingHuman = GetHumanById(human.Id);
+            if (existingHuman != null)
```

Comment:
W takich miejscach warto, zrobić condition pokroju if (existingHuman == null) throw new Exception(message: $"HUman with this id already exists.");